test/end-to-end.js

```diff
@@ -1043,6 +1043,63 @@ describe('end-to-end', function() {
         .end(done);
     }
   });
+
+  describe('Multi-project workspace support', function() {
+    beforeEach(givenEmptySandbox);
+    afterEach(function setWorkspaceToSandboxDir() {
+      process.env.WORKSPACE_DIR = SANDBOX;
+    });
+
+    it('switches workspace while creating two apps in the same dir',
+      function(done) {
+        async.series([
+          function(done) {
```

Comment:
maybe change all these internal dones to `next` and keep `done` as the main one on 1054 and 1100 for clarity.